repository.json

```diff
@@ -2119,6 +2119,19 @@
 					"tags": true
 				}
 			]
+		},
+		{
+			"name": "pycparser",
+			"description": "Complete C99 parser in pure Python",
+			"author": "Eli Bendersky",
+			"issues": "https://github.com/eliben/pycparser/issues",
+			"releases": [
+				{
+					"base": "https://github.com/eliben/pycparser",
```

Comment:
This base url will not work because a Github repository must follow a certain layout in order to be installable by PC, which this one does not. What you want instead is to refer to wheels on pypi. There are some examples of that in the file already.